olive/passes/onnx/vitis_ai_quantization.py

```diff
@@ -189,7 +227,7 @@ class VitisAIQuantization(Pass):
 
     def _initialize(self):
         super()._initialize()
-        self.tmp_dir = tempfile.TemporaryDirectory()
+        self.tmp_dir = tempfile.TemporaryDirectory(prefix="olive_vaiq_tmp")
 
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
```

Comment:
this parameter contains the ep info. when running, this parameter is Pass.accelerator_spec.